Broadcasting/BroadcastViewModel.swift

```diff
@@ -263,6 +263,42 @@ class BroadcastViewModel: NSObject, ObservableObject {
         }
     }
 
+    func willResignActive() {
+        guard sessionIsRunning,
+              let session = broadcastSession,
+              let cgImage = UIImage(named: "camera_off")?.cgImage else {
+                  return
+              }
+
+        // Create the background image source
+        guard let source = session.createAppBackgroundImageSource(withAttemptTrim: true, onComplete: { error in
```

Comment:
This API should only be invoked once (not every time the app goes to the background), and it needs to have to time finish in the foreground.

Because of those 2 things, it is recommended to set up this API immediately after the creation of your `IVSBroadcastSession`. That will give the API time to complete the encoding, and you can guarantee that it is only invoked once per session.

An additional benefit of calling this immediately is that it is an expensive API that can cause performance degradation if called while broadcasting.